src/utils/split_string.cpp

```diff
@@ -0,0 +1,21 @@
+//
+// Created by Piotr Moszkowicz on 2018-12-16.
+//
+
+#include "split_string.h"
+
+#include "pch.h"
+
+namespace Bald::Utils {
+    std::vector<std::string> split_string(const char* str_to_split, char delimiter) {
+        std::vector<std::string> result;
+        std::string temp;
+        auto str = static_cast<std::string>(str_to_split);
```

Comment:
instead  line 12 and 13 you could simply do std::string temp(str_to_split). std::string has a constructor which takes const char*.

Reply:
👍